### PR TITLE
Remove dead code prepare_environment in browser.py

### PR DESCRIPTION
**Pull request by foolip**
This isn't called from anywhere. The DBUS_SESSION_BUS_ADDRESS bits were
added in https://github.com/w3c/web-platform-tests/pull/5893 but was
unused even then. Since then it's been copied around a lot.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/9653)

### CODE REVIEW

**Comment by foolip** [comment]
Don't merge this even if reviewed. Needs to be rebased after https://github.com/w3c/web-platform-tests/pull/9649 is landed and update the documentation.

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-02-23 19:50:12
Finished: 2018-02-23 20:09:23





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24829)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/345390833)

**Comment by foolip** [comment]
Rebased and figured the documentation doesn't need to change. Ready for review.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**
